tests/class-test-cloudwatch-logs.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Altis\Cloud\Tests;
```

Comment:
Sorry what?

Reply:
Weird. I think you just add `phpcs:disable HM.Files.NamespaceDirectoryName.NameMismatch` for now. Seems like maybe a bug.

Reply:
I think this is from an old version of the coding standards that did some `substr()` wrong, I think we fixed this?